### DISCUSSION TITLE
Digitally signed PDF of student status

### DISCUSSION THREAD

**Discussion by numpde** [community/community#109718] [GitHub Education]
### Hi everyone,

I'd like to apply for the student developer pack. I have a PDF signed by my school confirming my student status. How to upload it?

**Comment by ettaboyle** [comment]
@numpde please check out the resources already documented and 📌 pinned in the [Education category](https://github.com/orgs/community/discussions/categories/github-education):

### [Student Developer Pack Application & FAQs ➡️ Start Here](https://github.com/orgs/community/discussions/17814)
